idd/Energy+.idd.in

```diff
@@ -546,15 +560,6 @@ ShadowCalculation,
        \key SimpleSkyDiffuseModeling
        \key DetailedSkyDiffuseModeling
        \default SimpleSkyDiffuseModeling
-  A4 , \field External Shading Calculation Method
-       \type choice
-       \key ScheduledShading
-       \key InternalCalculation
-       \key ImportedShading
```

Comment:
Are you proposing to remove this field "External Shading Calculation Method"? We need to keep it to import shading calculation results by external tools.

Reply:
@hongtz68 I'm not proposing to remove it. I'm proposing moving the capability to a new field: "[Shading Calculation Method](https://github.com/NREL/EnergyPlus/pull/7302/files/f3d0cab69b7f543b9db1fdc6d253e51c9eaeea26#diff-23ccf090b80d26e885712256b9a6d888R512)"